rediscluster/nodemanager.py

```diff
@@ -65,7 +67,7 @@ def random_node(self):
         return self.nodes[key]
 
     def get_redis_link(self, host, port, decode_responses=False):
-        return StrictRedis(host=host, port=port, decode_responses=decode_responses)
+        return StrictRedis(host=host, port=port, decode_responses=decode_responses, **self.connection_kwargs)
```

Comment:
The problem with your tests is here. The connection args that is sent in `test_repr_contains_db_info_unix` test should not be sent down to StrictRedis i guess. The connection_args that goes into ClusterConnectionPool is not the same as the args that is going into ClusterConnection class, there must be some way to seperate them.